VoxelSniperCore/src/main/java/com/github/kevindagame/voxelsniper/events/player/PlayerBrushSizeChangedEvent.java

```diff
@@ -8,16 +8,18 @@
 
 import java.util.function.Consumer;
 
+import org.jetbrains.annotations.NotNull;
+
 public class PlayerBrushSizeChangedEvent extends PlayerEvent<PlayerBrushSizeChangedEvent> implements Cancellable {
 
     private static final HandlerList<PlayerBrushSizeChangedEvent> handlers = new HandlerList<>();
     private final int oldSize;
     private final int newSize;
     private final String toolId;
-    private final IBrush brush;
+    private final @NotNull IBrush brush;
     private EventResult status;
 
-    public PlayerBrushSizeChangedEvent(IPlayer p, String toolId, IBrush brush, int oldSize, int newSize) {
+    public PlayerBrushSizeChangedEvent(IPlayer p, String toolId, @NotNull IBrush brush, int oldSize, int newSize) {
```

Comment:
Can a player change the brush size if no brush is assigned (due to permissions)?

Reply:
I made sure the event only gets called when a brush is assigned, didn't make much sense to me to change the size (and call the event) when no brush is assigned.